packages/ui/src/helpers/auth.ts

```diff
@@ -98,3 +102,32 @@ export const getActorState = (state: AuthMachineState): AuthActorState => {
 export const getActorContext = (state: AuthMachineState): AuthActorContext => {
   return getActorState(state)?.context;
 };
+
+/**
+ * Creates public facing auth helpers that abstracts out xstate implementation
+ * detail. Each framework implementation can export these helpers so that
+ * developers can send events without having to learn internals.
+ *
+ * ```
+ * const [state, send] = useActor(...);
+ * const { submit } = getSendAliases(send);
+ * submit({ username, password})
+ * ```
+ */
+export const getSendAliases = (send: Sender<AuthEvent>) => {
+  const sendToMachine = (type: AuthEventTypes) => {
+    return (data?: AuthEventData) => send({ type, data });
+  };
```

Comment:
`sendToMachine` creates alias for sending given event type to `authMachine`. `data` is optional, and if it's provided, it'll be sent with the event.

Reply:
For example `sendToMachine('CHANGE')` would return helper function

```ts
(data?: AuthEventData) => send({ type: 'CHANGE', data});
```